crds/operators.coreos.com_catalogsources.yaml

```diff
@@ -90,6 +91,11 @@ spec:
                   a registry-server with. Only used when SourceType = SourceTypeGrpc.
                   If present, the address field is ignored.
                 type: string
+              priority:
+                description: Priority represents weighting catalogs for the dependency
```

Comment:
Reword this. Priority field, if specified, is used to rank catalog sources that contains operators to supply dependent APIs and prioritize certain catalogs based on the ranking.